install/scripts/install-atarashi.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e 
+
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt-get update
+sudo apt-get install python>=3.6
+
```

Comment:
this installation script can be move to agent mod_deps file(not in its current form), as different distro has different types of supported version. 
You can use this script as a ref https://github.com/fossology/fossology/blob/master/src/ojo/mod_deps